### ISSUE TITLE
doesn't work on windows

### ISSUE THREAD

**Issue by Mavic129** [MAVProxyUser/P0VsRedHerring#2]
tar on windows doesn't support symlink
spawn is not available on windows

tested on windows 10

**Comment by HFMan** [comment]
hugh.. work from a fork.  When you are done, you can just do a pull request and this will notify MAVProxyUser he can pull your changes into master.

**Comment by MAVProxyUser** [comment]
https://github.com/MAVProxyUser/P0VsRedHerring/commits/cygwin-compatibility

Is this your commit @hugheaves ?

**Comment by hugheaves** [comment]
@MAVProxyUser, yes, that branch contains your script with some conditional logic to handle Cygwin. I'm guessing there's a native (non-Cygwin) ruby for Windows as well, but as the script made use of the typical unix commands, Cygwin seemed like the "path of least resistance".

P.S. Sorry for the late reply ... I just got back into town after a few days "on the road".

**Comment by MAVProxyUser** [comment]
I need to do some heavy testing, but I believe this patch cleans up windows. 
https://github.com/MAVProxyUser/P0VsRedHerring/commit/ac04a117cbea49dfcfecd216b2ed62d751cd03dd

**Comment by hugheaves** [comment]
Looks good. I think you can pretty much ignore my branch at this point, as I see that you've got the required cygwin binaries in master, (I was running using a full cygwin install, but that's overkill for this use case)
